assets/js/core.js

```diff
@@ -14,6 +14,55 @@ Author:         Suelo
 var $html = $('html');
 var $body = $('body');
 
+// Language selection trigger
+$('.language > ul > li > ul > li > a').on('click', (event) => {
+    event.preventDefault();
+    const selectedLang = $(event.target).html()
+    localStorage.setItem('userLanguage', selectedLang)
+    window.location.href = $(event.target).attr('href')
+})
+
+// Default Language selection
+const localStorage = window.localStorage
+let userLanguage = localStorage.getItem('userLanguage')
+
+let lang = window.navigator.languages ? window.navigator.languages[0] : null;
+lang = lang || window.navigator.language || window.navigator.browserLanguage || window.navigator.userLanguage;
+
+let shortLang = lang;
+if (shortLang.indexOf('-') !== -1)
+    shortLang = shortLang.split('-')[0];
+
+if (shortLang.indexOf('_') !== -1)
+    shortLang = shortLang.split('_')[0];
+
+const possibleLangs = { "pt": { "suffix": "BR", "fullName": "Português" }, 
+                        "sv": { "suffix": "SE", "fullName": "Svenska" },
+                        "en": { "suffix": "", "fullName": "English" }};
+
+ // If no language is selected, use browser language. If unsupported, use english.
+if (userLanguage === null) {
```

Comment:
Maybe you could put this check at line 28, since that code only need to be runned if this conditional is true. And the part inside the conditional could also be inside a function to retrieve the selectedLanguage. But this is not an impediment. :+1: LGTM